multimodal/src/autogluon/multimodal/utils/metric.py

```diff
@@ -43,9 +43,9 @@
 
 def infer_metrics(
     problem_type: Optional[str] = None,
-    pipeline: Optional[str] = None,
     eval_metric_name: Optional[str] = None,
     validation_metric_name: Optional[str] = None,
+    pipeline: Optional[str] = None,
```

Comment:
shall we remove this here as well?

Reply:
This is only used in `matcher`. I tried but failed to remove it in this PR so we may do it in a follow-up PR.